nf_core/utils.py

```diff
@@ -325,13 +325,21 @@ def fetch_wf_config(wf_path: Path, cache_config: bool = True) -> dict:
     result = run_cmd("nextflow", f"config -flat {wf_path}")
     if result is not None:
         nfconfig_raw, _ = result
-        for line in nfconfig_raw.splitlines():
-            ul = line.decode("utf-8")
-            try:
-                k, v = ul.split(" = ", 1)
-                config[k] = v.strip("'\"")
-            except ValueError:
-                log.debug(f"Couldn't find key=value config pair:\n  {ul}")
+        nfconfig = nfconfig_raw.decode("utf-8")
+        multiline_key_value_pattern = re.compile(r"(^|\n)([^\n=]+?)\s*=\s*((?:(?!\n[^\n=]+?\s*=).)*)", re.DOTALL)
+
+        for config_match in multiline_key_value_pattern.finditer(nfconfig):
+            k = config_match.group(2).strip()
+            v = config_match.group(3).strip().strip("'\"")
+            if k and v == "":
+                config[k] = ""  # or do we want to set it to "null"?
+                log.debug(f"Config key: {k}, value: empty string")
+            elif k and v:
+                config[k] = v
+                log.debug(f"Config key: {k}, value: {v}")
```

Comment:
are these to if-clauses doing the same thing in the end (just a slightly different debug message)?